runtime/compiler/control/JitDump.hpp

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+#include "bcnames.h"
+#include "jithash.h"
+#include "jitprotos.h"
+#include "j9.h"
+#include "j9cfg.h"
+#include "j9modron.h"
+#include "j9nonbuilder.h"
+#include "j9consts.h"
+#include "mmhook.h"
+#include "mmomrhook.h"
+#include "vmaccess.h"
+
+UDATA
+blankDumpSignalHandler(struct J9PortLibrary *portLibrary, U_32 gpType, void *gpInfo, void *arg);
+
+intptr_t
```

Comment:
Previously, this was marked `extern "C"`. Does it link now?

Reply:
Yes it links now. See all passing tests. I removed the extern declaration because the functions are never invoked via name, but are instead invoked via function tables and thus we don't need to declare them with the `extern "C"` qualifier.

Reply:
Actually I've reinstated the `exern "C"` qualifier. Fixed in af69312. See commit message for details.